include/LightGBM/c_api.h

```diff
@@ -324,293 +343,333 @@ LIGHTGBM_C_EXPORT int LGBM_DatasetSetField(DatasetHandle handle,
                                            int type);
 
 /*!
-* \brief get info vector from dataset
-* \param handle a instance of data matrix
-* \param field_name field name
-* \param out_len used to set result length
-* \param out_ptr pointer to the result
-* \param out_type  C_API_DTYPE_FLOAT32 or C_API_DTYPE_INT32
-* \return 0 when succeed, -1 when failure happens
+ * @fn LGBM_DatasetGetField
+ * @brief Get info vector from dataset.
+ * @param handle An instance of data matrix
+ * @param field_name Field name
+ * @param out_len Used to set result length
+ * @param out_ptr Pointer to the result
+ * @param out_type C_API_DTYPE_FLOAT32 or C_API_DTYPE_INT32
```

Comment:
` * @param[out] out_type Type of result pointer, can be C_API_DTYPE_INT8, C_API_DTYPE_INT32, C_API_DTYPE_FLOAT32 or C_API_DTYPE_FLOAT64`